File: 4.AHIFS/BSD[POS1]/Programme/004Judth_Blaschke_SessionRaterV2/SessionRaterJS/Rating.js
var nextID = 1;
function Rating(ratingValue, evaluator){
    var ratingID = nextID++;
    var ratingValue = ratingValue;
    var evaluator = evaluator;
    
    this.toString = toString;

    function toString(){
        return "ID: " + ratingID + ", evaluator: " + evaluator + ", rating-value: " + ratingValue;
    }
}

module.exports = Rating;

File: 4.AHIFS/BSD[POS1]/Programme/003Judth_Blaschke_SessionRater/SessionRaterClient/Rating.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SessionRaterClient
{
    public class Rating
    {
        public int RatingId { get; }
        public int RatingValue { get; set; }
        public string Evaluator { get; set; }
        public Session EvaluatedSession { get; set; }
        private static int nextId = 1;

        public Rating(int RatingValue, string Evaluator, Session EvaluatedSession)
        {
            RatingId = nextId;
            nextId++;
            this.Evaluator = Evaluator;
            this.EvaluatedSession = EvaluatedSession;
            this.RatingValue = RatingValue;
        }
    }
}